docs/api/rest_api/rest_api_reference/input/examples/shipments/identifier/PATCH/ShipmentUpdate.json.example

```diff
@@ -0,0 +1,14 @@
+{
+    "ShipmentUpdate": {
+        "identifier": "new_identifier",
+        "transition": "prepare",
+        "Owner": {
+            "_media-type": "application/vnd.ibexa.api.User+json",
+            "_href": "/api/ibexa/v2/user/users/14"
+        },
+        "Context": {
```

Comment:
TODO: How this is reflected on resulting …/[PATCH/Shipment.json.example](https://github.com/ibexa/documentation-developer/pull/2053/files#diff-a95c7ea0c13ef731ef50d859642e19d9e8356e4839076bcdcdd9090923d0185a)?

Reply:
It seems there is no visitor to have a shipment context in this output.